playbooks/roles/logstash/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+logstash_apt_repo_url:
+"http://packages.elasticsearch.org/logstash/1.4/debian"
+
+logstash_apt_repos:
+  - { repo: "deb {{ logstash_apt_repo_url }} stable main", state: "present" }
+
+logstash_apt_keys:
+  - { url: "http://packages.elasticsearch.org/GPG-KEY-elasticsearch", state: "present" }
+
+logstash_apt_packages:
+  - logstash
+  - logstash-contrib
+  - openjdk-7-jre
+
+logstash_pip_packages:
+  - python-memcached
```

Comment:
Are these pip packages actually required? To my knowledge neither are actually used.